test/helper/config_tarantool.lua

```diff
@@ -0,0 +1,188 @@
+    src:close()
+    return yaml.decode(data)
+end
+
+local function write_config(path, decoded)
+    local dst = fio.open(path, {'O_CREAT', 'O_WRONLY', 'O_TRUNC'})
+    dst:write(yaml.encode(decoded))
+    dst:close()
+end
+
+function config.get_cfg()
+    local decoded = read_config(config.devconfigpath)
+
+    if decoded['roles_cfg'] ~= nil and decoded['roles_cfg'][roles[1]] then
+        return decoded['roles_cfg'][roles[1]]['cfg']
+    end
+    return nil
+end
+
+function config.set_cfg(cfg)
+    local decoded = read_config(config.devconfigpath)
+
+    if decoded['roles_cfg'] == nil then
+        decoded['roles_cfg'] = {}
+    end
+    for _, role in ipairs(roles) do
+        if decoded['roles_cfg'][role] == nil then
+            decoded['roles_cfg'][role] = {}
+        end
+        decoded['roles_cfg'][role]['cfg'] = cfg
+    end
+
+    write_config(config.devconfigpath, decoded)
+
+    config.reload()
+end
+
+function config.create_tube(tube_name, options)
+    options = options or {}
+
+    local decoded = read_config(config.devconfigpath)
+
+    if decoded['roles_cfg'] == nil then
+        decoded['roles_cfg'] = {}
+    end
+    for _, role in ipairs(roles) do
+        if decoded['roles_cfg'][role] == nil then
+            decoded['roles_cfg'][role] = {}
+        end
+        if decoded['roles_cfg'][role]['tubes'] == nil then
+            decoded['roles_cfg'][role]['tubes'] = {}
+        end
+        decoded['roles_cfg'][role]['tubes'][tube_name] = {}
+        for k, v in pairs(options or {}) do
+            decoded['roles_cfg'][role]['tubes'][tube_name][k] = v
+        end
+    end
+
+    write_config(config.devconfigpath, decoded)
+
+    config.reload()
+end
+
+function config.drop_tube(tube_name)
+    local decoded = read_config(config.devconfigpath)
+
+    if decoded['roles_cfg'] == nil then
+        return
+    end
+    for _, role in ipairs(roles) do
+        if decoded['roles_cfg'][role] == nil then
+            return
+        end
+        if decoded['roles_cfg'][role]['tubes'] == nil then
+            return
+        end
+        if decoded['roles_cfg'][role]['tubes'][tube_name] == nil then
+            return
+        end
+        decoded['roles_cfg'][role]['tubes'][tube_name] = nil
+    end
+
+    write_config(config.devconfigpath, decoded)
+
+    config.reload()
+end
+
+function config.eval(server_name, ...)
+    return config.servers[server_name]:eval(...)
+end
+
+function config.get_evaler(server_name)
+    return config.servers[server_name]
+end
+
+t.before_suite(function()
+    fio.rmtree(config.datadir)
+    fio.mktree(config.datadir)
+
+    fio.copyfile(config.configpath, config.devconfigpath)
+    fio.copytree(fio.pathjoin(config.root, 'roles'),
+        fio.pathjoin(config.datadir, 'roles'))
+    fio.copytree(fio.pathjoin(config.root, 'sharded_queue'),
+        fio.pathjoin(config.datadir, 'sharded_queue'))
+
+    config.start_example_replicaset()
+
+    fio.mktree(config.unitdir)
+    box.cfg{
```

Comment:
Setting up box.cfg for a luatest test execution server is a bad practice. Why do we need this?

Reply:
Removed.